com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -354,75 +428,98 @@ private void SendSceneEventData(ulong[] targetClientIds)
         }
 
         /// <summary>
-        /// Returns the Netcode scene index from a scene
+        /// Verifies the scene name is valid relative to the scenes in build list
         /// </summary>
-        /// <param name="scene"></param>
-        /// <returns>Netcode Scene Index</returns>
-        private uint GetNetcodeSceneIndexFromScene(Scene scene)
+        /// <param name="sceneName"></param>
+        /// <returns>true (Valid) or false (Invalid)</returns>
+        internal bool IsSceneNameValid(string sceneName)
```

Comment:
I think I would combine this with `GetBuildIndexFromSceneName` and just have the caller check on `uint.MaxValue`.  There are places later in the code that this I think would make more readable